sdk/core/amqp-common/changelog.md

```diff
@@ -1,5 +1,9 @@
 ### 2019-5-6 1.0.0-preview.5
 
+- Includes bug fix for compatibility issue with using MSI credentials. [PR 4098](https://github.com/Azure/azure-sdk-for-js/pull/4098)
```

Comment:
The changelog needs to be clearer.  It should be able to convey the issue that was fixed at a glance.